styles/colors-silicon-night.less

```diff
@@ -7,6 +7,46 @@
 // as these are overrides to the "base" layer of colors.
 //
 
+// Silicon skin color palette
+@silicon-white: #ffffff;
+@silicon-black: #000000;
+@silicon-red-pink: #f1304f;
+@silicon-dark: #1b1f35;
+@silicon-vibrant-purple: #9e00d8;
+@silicon-magenta: #ba0088;
+@silicon-blush: #f2acac;
+@silicon-sky: #88d2fc;
+@silicon-foam-green: #8afcb0;
+@silicon-purply-blue: #6b24f2;
+@silicon-medium-pink: #ea448e;
+@silicon-faded-orange: #f3ae4d;
+@silicon-dark-two: #2a2742;
+@silicon-gradient: linear-gradient(to right, @silicon-vibrant-purple, @silicon-magenta, @silicon-red-pink);
+
+
+// Abstracted Colors
+@agate-accent: var(--agate-accent-color, @silicon-red-pink);
+@agate-accent-h: var(--agate-accent-h, 350);
+@agate-accent-s: var(--agate-accent-s, 78%);
+@agate-accent-l: var(--agate-accent-l, 57%);
+@agate-highlight: var(--agate-highlight-color, @silicon-vibrant-purple);
+@agate-highlight-h: var(--agate-highlight-h, 284);
+@agate-highlight-s: var(--agate-highlight-s, 100%);
+@agate-highlight-l: var(--agate-highlight-l, 42%);
+@agate-background: white;
+@agate-foreground: black;
+@agate-background-active: fade(black, 15%);
+@agate-foreground-active: white;
+@agate-background-focus: @agate-accent;
+@agate-foreground-focus: white;
+@agate-gradient:    @silicon-gradient;
+@agate-black:       @silicon-black;
+@agate-darker-gray: @silicon-dark;
+@agate-dark-gray:   @silicon-dark-two;
+@agate-gray:        #8a8a8a;
+@agate-light-gray:  #ededed;
+@agate-white:       @silicon-white;
+
```

Comment:
This is unnecessary, as these colors are already defined in the day skin, which is the base for this skin. These are all available automatically without redefining them here.

Here's the relevant piece of code that loads the day values into scope, and then the night values on top of those. (Found in `skin.less`)
```css
	.buildSkinRules(silicon; {
		@import (multiple) "./variables-silicon.less";

		@import (multiple) "./colors-silicon-day.less";
		@componentRules();

		.buildSkinRules(night; {
			@import (multiple) "./colors-silicon-night.less";
			@componentRules();
		});
	});
```